ppb/features/default_sprites.py

```diff
@@ -58,3 +60,118 @@ def _exponential_decay(self, time_delta):
         remaining = decay_rate ** time_delta
         decay_amount = 1. - remaining
         return decay_amount
+
+
+@dataclass
+class DirectionKeys:
+    up: ppb.keycodes
+    left: ppb.keycodes
+    down: ppb.keycodes
+    right: ppb.keycodes
+
+
+@dataclass
+class ArrowDirectionKeys(DirectionKeys):
+    up: ppb.keycodes = ppb.keycodes.Up
+    left: ppb.keycodes = ppb.keycodes.Left
+    down: ppb.keycodes = ppb.keycodes.Down
+    right: ppb.keycodes = ppb.keycodes.Right
+
+
+@dataclass
+class WASDDirectionKeys(DirectionKeys):
+    up: ppb.keycodes = ppb.keycodes.W
+    left: ppb.keycodes = ppb.keycodes.A
+    down: ppb.keycodes = ppb.keycodes.S
+    right: ppb.keycodes = ppb.keycodes.D
+
+
+class KeyboardMovementSprite(ppb.Sprite):
+    """Sprite that moves with keyboard movement keys.
+
+    :param direction_keys: Configuration for mapping direction to keys
+    :param direction: Starting direction
+    :param acceleration: The increase in speed for a repeated key press in the following 0.3 seconds.
+    :param friction: The decrease in speed when there are no key presses in the following 0.3 seconds.
+    :param initial_speed: The speed per second when the first of subsequent keypresses are made.
+    :param speed: The current speed per second.
+    :param max_speed: Maximum speed per second that the sprite can travel in a specific direction.
+    :param min_speed: Minimum speed per second that the sprite can travel in a specific direction.
+    :param last_key_pressed_at: The datetime of the last time any key was pressed.
```

Comment:
This isn't really a user-facing parameter.

Reply:
good point, will remove